Turbo Frame

Mildred14 opened the pull request.

Mildred14 committed:
Add tubro frame tags for messages on edit and new

danny-rosenblatt-salesloft commented on app/javascript/packs/application.js:
```diff
@@ -13,3 +13,5 @@ require("@fortawesome/fontawesome-free/js/all");
 Turbolinks.start()
 ActiveStorage.start()
+
+import "controllers"
```
Try adding
```js
import "@hotwired/turbo-rails"
```

Mildred14 committed:
Add turbo
Use turbo frame tag
Edit messages in boards
Add css details for tables and board
